lib/util/git/index.ts

```diff
@@ -165,13 +165,8 @@ export async function validateGitVersion(): Promise<boolean> {
   let version: string | undefined;
   const globalGit = simpleGit();
   try {
-    const raw = await globalGit.raw(['--version']);
-    for (const section of raw.split(/\s+/)) {
-      if (semverCoerced.isVersion(section)) {
-        version = section;
-        break;
-      }
-    }
+    const { major, minor, patch } = await globalGit.version();
+    version = `${String(major)}.${String(minor)}.${String(patch)}`;
```

Comment:
```suggestion
    version = `${major}.${minor}.${patch}`;
```

why those string stuff?

Reply:
TS string literal error.

Reply:
Inferred type for major, minor and patch are bumber only.Issue isn't for null. 
It is that inside template literal expression we cannot use variable which is other than string type.

Reply:
number should be ok. 🤔

Reply:
It doesn't throw the error now🙂. Sometimes it does and other times it doesn't ...weird.